src/Microsoft.SqlTools.ServiceLayer/ShowPlan/ExecutionPlanService.cs

```diff
@@ -49,6 +50,23 @@ internal IProtocolEndpoint ServiceHost
         public void InitializeService(ServiceHost serviceHost)
         {
             this.ServiceHost = serviceHost;
+            this.ServiceHost.SetRequestHandler(GetExecutionPlanRequest.Type, HandleGetExecutionPlan);
+        }
+
+        private async Task HandleGetExecutionPlan(GetExecutionPlanParams requestParams, RequestContext<GetExecutionPlanResult> requestContext)
+        {
+            try
+            {
+                var plans = ShowPlanGraphUtils.CreateShowPlanGraph(requestParams.GraphFile.GraphFileContent, "");
+                await requestContext.SendResult(new GetExecutionPlanResult
```

Comment:
do you need to set `ResultStatus.Success` to `true` (or not derive the return value from `ResultStatus`)?